be/src/vec/functions/array/function_array_element.h

```diff
@@ -356,6 +400,15 @@ class FunctionArrayElement : public IFunction {
         } else if (check_column<ColumnString>(*nested_column)) {
             res = _execute_string(offsets, *nested_column, src_null_map, *idx_col, nested_null_map,
                                   dst_null_map);
+        } else if (check_column<ColumnArray>(*nested_column)) {
+            res = _get_mapped_value<ColumnArray>(offsets, *nested_column, src_null_map, *idx_col,
```

Comment:
1. change name to _execute_common
2. impl to type awareless

Reply:
done!